tensorpack/callbacks/param.py

```diff
@@ -151,12 +151,10 @@ def get_value_to_set(self):
         """
         ret = self._get_value_to_set()
         if ret is not None and ret != self._last_value:
-            if self.epoch_num != self._last_epoch_set:
-                # Print this message at most once every epoch
-                logger.info("[HyperParamSetter] At global_step={}, {} will change to {:.8f}".format(
-                    self.global_step, self.param.readable_name, ret))
+            logger.info("[HyperParamSetter] At global_step={}, {} will change to {:.8f}".format(
```

Comment:
I tried to avoid printing this information too often because of how noisy it can be. But I understand there is a confusion when it's only printed once per epoch. Maybe the message or the docs can be improved.

Reply:
OK, I understand..

Reply:
Do you want me to keep this change or revert this change in order for you future change on this part?